app.py

```diff
@@ -1,16 +1,15 @@
-from . import create_app
-from . import db
+from gateway import create_app
+from gateway import db
 
 # Call the Application Factory function to construct a Flask application instance
 # using the standard configuration defined in /instance/flask.cfg
 app = create_app('flask.cfg')
-
-
-@app.before_first_request
-def create_tables():
+with app.app_context():
     db.create_all()
```

Comment:
I think this should be done in the `gateway.create_app()` function. Any reason to put it outside the Flask app setup codes?

Reply:
I could not get it working when it was inside the create_app function. 
Would be good to understand more about context.